lib/pwwka/transmitter.rb

```diff
@@ -23,10 +23,17 @@ class Transmitter
 
     DEFAULT_DELAY_BY_MS = 5000
 
+    attr_reader :app_manages_connector
```

Comment:
Nitpick: could we call this `caller_manages_connector`?   There might/will be many callers within an application, and they may not al use this behaviour.

Reply:
https://github.com/stitchfix/pwwka/pull/98/commits/6c77929156cc7c6f153f7f8cb162e93ac32894bf